pkg/sql/catalog/descs/collection.go

```diff
@@ -50,8 +50,8 @@ func makeCollection(
 	virtualSchemas catalog.VirtualSchemas,
 	temporarySchemaProvider TemporarySchemaProvider,
 	monitor *mon.BytesMonitor,
-) Collection {
-	return Collection{
+) *Collection {
```

Comment:
the `make` terminology is differentiated from the `new` terminology in this codebase and other to indicate that it's returning a struct and not a pointer. You need to rename this `newCollection`.

Reply:
Done